src/cozmo/run.py

```diff
@@ -499,12 +499,12 @@ def _connect_async(f, conn_factory=conn.CozmoConnection, connector=None):
         loop.run_forever()
 
 
+_sync_loop = asyncio.new_event_loop()
```

Comment:
is it bad that this is always created, even for async programs that won't use it?